ta4j-core/src/main/java/org/ta4j/core/Order.java

```diff
@@ -251,4 +251,8 @@ public static Order sellAt(int index, Num price, Num amount) {
     public static Order sellAt(int index, TimeSeries series, Num amount) {
         return new Order(index, series, OrderType.SELL, amount);
     }
+
```

Comment:
Please add javadoc for Order#getValue method

Reply:
fixed